build/package/darwin/newrelic-infra.plist

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
isn't the same? https://github.com/newrelic/infrastructure-agent/blob/master/build/package/launchd/Library/LaunchDaemons/com.newrelic.infrastructure.plist

Reply:
embedded in the brew formula